test/integration/log_system.cc

```diff
@@ -1523,7 +1523,7 @@ TEST_F(LogSystemTest, IGN_UTILS_TEST_DISABLED_ON_WIN32(LogResources))
   // Recorded models should exist
   EXPECT_GT(entryCount(recordPath), 2);
   EXPECT_TRUE(common::exists(common::joinPaths(recordPath, homeFake,
-      ".ignition", "fuel", "fuel.ignitionrobotics.org", "openrobotics",
+      ".gz", "fuel", "fuel.ignitionrobotics.org", "openrobotics",
```

Comment:
We shouldn't change these yet

Reply:
[`a3ecb2b` (#1520)](https://github.com/gazebosim/gz-sim/pull/1520/commits/a3ecb2b9b170b743a89c459faba90884b22eb70d)